scripts/ci-workflow

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+
+./build
+./validate
+./test -cover
```

Comment:
Can we reuse the scripts/ci and add a new parameter like `workflow`?
If `workflow` is `true`, then no need to run package.

Reply:
@COLDTURNIP Let's remove this file. Thank you.